app/helpers/perilune/home_helper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Perilune
+  module HomeHelper
+    def get_general_success_and_failure_series(type:, from:, to:)
+      success_series = get_general_series_for(
+        type: type, from: from,
+        to: to, status: 'success'
+      )
+
+      failure_series = get_general_series_for(
+        type: type, from: from,
+        to: to, status: 'failure'
+      )
+
+      [success_series, failure_series]
+    end
```

Comment:
Method called from front.
Here we receive the parameter needed for Trifle gem:
- type: `import` / `export`
-  from / to

With this data, we call a private method who'll use Trifle gem and do the request. Let's switch to the private method.